test/integration/ae_mdw_web/controllers/tx_controller_test.exs

```diff
@@ -1366,33 +1366,20 @@ defmodule Integration.AeMdwWeb.TxControllerTest do
       end)
 
       assert Enum.any?(response["data"], fn %{"tx" => block_tx} ->
-               if block_tx["type"] == "GAMetaTx" do
-                 block_tx["tx"]["tx"]["recipient_id"] == id
-               else
-                 false
-               end
+               block_tx["type"] == "GAMetaTx" && block_tx["tx"]["tx"]["recipient_id"] == id
              end)
 
       assert Enum.any?(response["data"], fn %{"tx" => block_tx} ->
-               if block_tx["type"] == "GAMetaTx" do
-                 block_tx["tx"]["tx"]["sender_id"] == id
-               else
-                 false
-               end
+               block_tx["type"] == "GAMetaTx" && block_tx["tx"]["tx"]["sender_id"] == id
              end)
 
       assert Enum.any?(response["data"], fn %{"tx" => block_tx} ->
-               if block_tx["type"] == "SpendTx" do
-                 assert block_tx["recipient_id"] == id or block_tx["sender_id"] == id
-                 true
-               else
-                 false
-               end
+               block_tx["type"] == "SpendTx"
```

Comment:
missing only the `and (block_tx["recipient_id"] == id or block_tx["sender_id"] == id)`
to validate the account

Reply:
Nice catch, not sure why I removed it.